benchmarks/CMakeLists.txt

```diff
@@ -1,9 +1,9 @@
-set(CMAKE_C_FLAGS "${CMAKE_C_FLAGS} -Wall -O2")
+set(CMAKE_C_FLAGS "${CMAKE_C_FLAGS} -Wall -O3")
 
 if (MSVC)
-    set(CMAKE_CXX_FLAGS "${CMAKE_CXX_FLAGS} -std:c++20 -O2")
+    set(CMAKE_CXX_FLAGS "${CMAKE_CXX_FLAGS} -std:c++20 -O3")
```

Comment:
This is wrong - MSVC does not support `-O3`. See [`/O` options (Optimize code)](https://learn.microsoft.com/en-us/cpp/build/reference/o-options-optimize-code?view=msvc-170), `-O2` is the highest option that MSVC supports.